docs/resiliency/index.asciidoc

```diff
@@ -152,6 +152,19 @@ We have ported the known scenarios in the Jepsen blogs that check loss of acknow
 The new tests are run continuously in our testing farm and are passing. We are also working on running Jepsen independently to verify
 that no failures are found.
 
+[float]
+=== Replicas can fall out of sync when a primary shard fails (STATUS: ONGOING)
+
+In a scenario with more than one replica and a primary shard fails, one of the
```

Comment:
Is this OK like it is? you're the english master but it feels weird to me. Maybe rewrite these two sentences to "When a primary fails, one of the replicas will be promoted to be the primary shard. If there are more than one replicas, it is possible for the remaining...."